### PR TITLE
Adds documentation for log prints functionality

### PR DESCRIPTION
**Pull request by WillRaphaelson**
### Checklist

- [x] This pull request references any related issue by including "closes `<link to issue>`"
	- If no issue exists and your change is not a small fix, please [create an issue](https://github.com/PrefectHQ/prefect/issues/new/choose) first.
- [x] This pull request includes tests or only affects documentation.
- [ ] This pull request includes a label categorizing the change e.g. `fix`, `feature`, `enhancement`

### CODE REVIEW

File: docs/concepts/logs.md [hunk 1]
```diff
@@ -125,6 +125,42 @@ Prefect automatically uses the task run logger based on the task context. The de
 
 The underlying log model for task runs captures the task name, task run ID, and parent flow run ID, which are persisted to the database for reporting and may also be used in custom message formatting.
 
+### Logging print statements
+
+For prototyping or local debugging, Prefect provides a decorator kwarg `log_print=True` to log print statements at the task or flow level. Tasks and subflows will inherit the print logging behavior from their parent flow.
```
Comment (madkinsz) [inline docs/concepts/logs.md:130]:
```suggestion
For prototyping or local debugging, Prefect provides the `log_prints` option to enabling logging of print statements at the task or flow level. By default, tasks and subflows will inherit the print logging behavior from their parent flow.
```

Do we want to have the prototyping/debugging note?

Comment (tpdorsey) [reply docs/concepts/logs.md:130]: "to enable logging of" I think?

File: docs/concepts/logs.md [hunk 2]
```diff
@@ -125,6 +125,42 @@ Prefect automatically uses the task run logger based on the task context. The de
 
 The underlying log model for task runs captures the task name, task run ID, and parent flow run ID, which are persisted to the database for reporting and may also be used in custom message formatting.
 
+### Logging print statements
+
+Prefect provides the `log_prints` option to enabling logging of print statements at the task or flow level. By default, tasks and subflows will inherit the print logging behavior from their parent flow.
```
Comment (madkinsz) [inline docs/concepts/logs.md:130]:
```suggestion
Prefect provides the `log_prints` option to enable logging of print statements at the task or flow level. By default, tasks and subflows will inherit the print logging behavior from their parent flow.
```

Sorry about the typo :)

File: docs/concepts/logs.md [hunk 3]
```diff
@@ -127,7 +127,7 @@ The underlying log model for task runs captures the task name, task run ID, and
 
 ### Logging print statements
 
-Prefect provides the `log_prints` option to enable logging of print statements at the task or flow level. By default, tasks and subflows will inherit the print logging behavior from their parent flow.
+Prefect provides the `log_prints` option to enabling logging of print statements at the task or flow level. By default, tasks and subflows will inherit the print logging behavior from their parent flow, unless opted out with their own `log_print=False`.
```
Comment (madkinsz) [inline docs/concepts/logs.md:130]: (lol)

```suggestion
Prefect provides the `log_prints` option to enable logging of print statements at the task or flow level. By default, tasks and subflows will inherit the print logging behavior from their parent flow, unless opted out with their own `log_print=False`.
```

File: docs/concepts/logs.md [hunk 4]
```diff
@@ -152,6 +152,31 @@ Will output:
...
+Using `log_prints=False` at the task level will output:
+
+<div class='terminal'>
+```bash
+14:18:57.401 | INFO    | prefect.engine - Created flow run 'singing-malkoha' for flow 'my-flow'
+14:18:58.026 | INFO    | Flow run 'singing-malkoha' - we're logging print statements from a flow
+14:18:58.148 | INFO    | Flow run 'singing-malkoha' - Created task run 'my_task-20c6ece6-0' for task 'my_task'
+14:18:58.150 | INFO    | Flow run 'singing-malkoha' - Executing 'my_task-20c6ece6-0' immediately...
+```
+</div>
```
Comment (madkinsz) [inline docs/concepts/logs.md:179]: The task print should still happen, just not be sent to a Prefect logger.

**Review by madkinsz — APPROVED** [review]

**Pull request merged**